google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -235,6 +235,7 @@ protected void textChanged(DocumentEvent event) {
         repositorySelector.setCloudProject(projectSelector.getText());
         repositorySelector.setUser(projectSelector.getSelectedUser());
         repositorySelector.setText("");
+        repositorySelector.loadRepositories(null /*onComplete*/);
```

Comment:
Didn't see the Goetz pres. yet, but plan to. Many of the places I've randomly read online talk about it being bad practice. E.g.:

http://stackoverflow.com/questions/31922866/why-should-java-8s-optional-not-be-used-in-arguments
https://dzone.com/articles/java-8-optional-how-use-it

But they could be totally wrong. I need to read more about it.